main.py

```diff
@@ -1,10 +1,45 @@
 import init_django_orm  # noqa: F401
 
-from db.models import Race, Skill, Player, Guild
+import json
+
+from db.models import Race, Skill, Guild, Player
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as f:
+        players_data = json.load(f)
+
+    for player_name, player_data in players_data.items():
+        race_data = player_data["race"]
+        race, _ = Race.objects.get_or_create(
+            name=race_data["name"],
+            description=race_data.get("description")
+        )
+
+        guild_data = player_data.get("guild")
+        guild = None
+        if guild_data:
+            guild, _ = Guild.objects.get_or_create(
+                name=guild_data["name"],
+                description=guild_data.get("description")
+            )
+
+        nickname = player_name
+        Player.objects.create(
+            nickname=nickname,
```

Comment:
Please pass `player_name `as an argument for the `nickname `parameter to avoid unnecessary variable creation.